.github/workflows/ci.yml

```diff
@@ -199,7 +188,6 @@ jobs:
         if: matrix.deploy
         with:
           target: ${{ matrix.target }}
-          image: ${{ steps.build-docker-image.outputs.image }}
```

Comment:
Just confirming that this is never used, correct? We don't use `image` prior, just `target`, as `${{ inputs.target }}`?

Reply:
yes, the only input is target: https://github.com/Emilgardis/cross/blob/eeae4f6ead3ce44cb687b126541e70bc7649e1a9/.github/actions/cargo-install-upload-artifacts/action.yml#L3-L6